torch/testing/_internal/common_methods_invocations.py

```diff
@@ -18844,6 +18844,7 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
     ),
     PythonRefInfo(
         "_refs.normal",
+        op=torch.Tensor.normal_,
```

Comment:
This doesn't seem right, as the `_refs.normal` models `torch.normal`

Reply:
It is using the entry from the op_db for normal_. I assumed that's why it was marked as in_place on the next line. I guess it could also be fixed by removing the `torch_opinfo_variant_name` that felt to me like a more major change. Perhaps there should be two tests here a "_refs.normal" and a "_refs.nomal_"?

Reply:
Note that there are many `OpInfo` variants of `normal` in that file. There's `in_place`, `number_mean` and one without variant. We should probably map all these to their own `PythonRefInfo`.

Reply:
I'm not really sure what number_mean is trying to test that the one without a variant hasn't got covered. It looks like the 3rd case of the non-variant test covers this case albeit with different inputs. There aren't tests for normal(float, float, shape) or normal(float, tensor) perhaps it was trying to test on of those

Reply:
Yeah, that seems odd. Feel free to send another PR deleting that variant and tag me as a reviewer, if you want.

Reply:
I'll change the number_mean variant to cover the `normal(float, float, shape)` case

Reply:
I see - the case where the fist parameter is a number needs to be separated out